### PR TITLE
API-6463: Syncing Organization records with production cdw

### PR DESCRIPTION
**Pull request by jabando-va**

### CODE REVIEW

**Review by nmoisan — COMMENTED** [review]
Just a couple of comments if the intent is to match what production data will look like.

File: datamart/dm-records-1017283180V801730/dmOrg457959I.json [hunk 1]
```diff
@@ -1,37 +1,14 @@
...
-  "objectType" : "Organization",
-  "objectVersion" : "1",
-  "cdwId" : "457959:I",
-  "stationIdentifier" : "561A4",
-  "npi" : "7112784910",
-  "providerId" : "0000000000300",
-  "ediId" : "26152",
-  "agencyId" : "other",
-  "active" : true,
-  "type" : {
```
Comment (nmoisan) [inline datamart/dm-records-1017283180V801730/dmOrg457959I.json:11]: Guess this comment wasn't commited.

this field will also not be null as a whole. `Display` might possibly be NULL, but `system` and `code` will always exist.

File: datamart/dm-records-1017283180V801730/dmOrg457959I.json [hunk 2]
```diff
@@ -1,37 +1,14 @@
-{
-  "objectType" : "Organization",
-  "objectVersion" : "1",
-  "cdwId" : "457959:I",
-  "stationIdentifier" : "561A4",
```
[the remark this answers is not in the document]
Comment (jabando-va) [reply datamart/dm-records-1017283180V801730/dmOrg457959I.json:5]: understood, what would the value for this field be for the Organization LYONS- VA NEW JERSEY HCS?

[the remark this answers is not in the document]
Comment (nmoisan) [reply datamart/dm-records-1017283180V801730/dmOrg457959I.json:5]: "stationIdentifier": "544"

File: datamart/dm-records-1017283180V801730/dmOrg457959I.json (hunk 1 above)
Comment (jabando-va) [reply datamart/dm-records-1017283180V801730/dmOrg457959I.json:11]: What would the type value be for LYONS- VA NEW JERSEY HCS? Would it be `"type" : {
    "system" : "institution",
    "code" : "CBOC",
    "display" : "COMMUNITY BASED OUTPATIENT CLINIC"
  }`?

Comment (nmoisan) [reply datamart/dm-records-1017283180V801730/dmOrg457959I.json:11]: "type":{"system":"Institution","code":"VAMC","display":"VA MEDICAL CENTER"} is what I am seeing.

**Comment by nmoisan** [comment]
This data now matches CDW Production data. Thank you JJ

**Review by jhulbert-va — APPROVED** [review]

**Review by eclendenning-va — APPROVED** [review]

**Pull request merged**